slides/developer-guide/working-with-shapes/adding-a-picture-to-a-presentation-slide/_index.md

```diff
@@ -0,0 +1,445 @@
jsez+o6tu12sbz9zrf/3niy17b9EKr00XVdYc3jR6eI5AMgYCqeB29K05DDfGJKC7vVAqbje3bdB3yA/f3jvg+bZm3MjJmYCRFZyGhbHgEAmGEyntwvlzEGYwyzHmOl3owiMJvuLZZLRDg+Pru6es4cq2oGgGVZA8DV9fnl1Utni3o0KYpqf+9wOju0thjGue1HH/0EAPLhIiqWTwFL1V+QSBVBxKoaFUXx+mvv6FFGA+/BMXZ9W1hSS+mAYDrESyAmh2LmdMbSAD45kchDEZvaViJaPDw4mg90dHTaPN92XZtan2QgeagyoiACztmyLIIPbIt8LCwwgzEQGUb1uCzKGON4PNvfP769uUDUvpwgorWWWXzwdy9/07br+fxEz2JQUR8i2ru7y911JSJrTa7+RAvADHap65o3Xn93NJ42TZ//2gkzsw+eEM5OTwgpZI2U3kdkkCRUHBTnwzDkUE5z3oaoakKRKJJOjM79i2isOTo4Pr96pUfsMTMRCIuYROAhCAKOquq+X+3w7pB6LALWmrqqN82WmZ88eXu9XoqwYoDMh8S+b0Poj4+e6p8fiDGGsNYkbpXqSz8pwpF3z8zRDg+iHnkiRVE+OnstBNmxVGLTnpyeGrIhiRDTpmAYFHi8O2I/eHEIfQhRhbmqjM7yKGetIbIiiUETQR94NJ6V9zd931qrg9lRlJAWRATddWVRiCT9sjEqgNOIKQBYj0ZqrNFoPJ3urVb3Cs00YoYYN5vFZDxHIj1FR3Ou1vNWjzbxfRujF44AiETWFq4onausIU4tI4ox7u8fT8Z7eu4kc0Ilfd8ezOd1VSqlrbiJBXJZzMNMDHOIMbRtq+MVRGQMOmd19G+zWYlwWdaj0czawFxYq9yAHrGpIAwmk9n1zVVVjZijiIVURSZqRES74sTCNgPggdcAgKqsEDWT0P7+yWp1p3tKV7RtVtbYshqpuoIZRKI6U4zBLpc3vltb46yrh0P2Y/TbdQOIVTmu6rGOozDz0eEjSK8b8yiML4tif74XIuBgKU6WGqroEALH0HZN1zWFc8eHR6PRxLkSAKMAR2FhPW/+4uL5avXV3t5RXY8l/VEJyFEfIkM9msLNVQje2iIhjd0j1gWUeNNmdT7FT4RBD+rVVoOmptnsQEc8dJfFGLu+mU72tc6WNG8o+WhXtuI3s/lJUU5SNYGICAioupq22XRdM57MVZy6t3+iYDdvv8jRHx6eDtBDMhfMIpJ3XIw+xLDdrizR40dnk/EUESJDjPpnsQJzUjzs75/u7Z2cn//2xYvPDg5Ox+MZZBYUEQEsMzvrrKEQfD5FLe36h84fIRHF5FGIQ68PVL1jjLqrcFnWVTXabFbGGObYdy0KuKISZkgFnLBw8L33/XZ9Y2cHryGiSAwh7kA+JMKqGpfVqG02m/WiL7pHj16rqpH3YYjN3vfjuh7VlY9aTgIzRAZOg33KN3jvu6bZzCbT46MjRRUiElMGSMhDE673HSI9fvwNY+xvfvN3uayxWY8sAIJExlqt0jOAzTD2YcMR5NFKSQfOgfZKDeHwN4GsdePxfLW6JyIR8aEzNskJ8jAqh9D3vt/cv6rK0up5SvrQrES0S1FCXU/Lsr66elm4wpDppeedM17ms9nwx4hEIKZRqxSnYgze902zPT48Otib9z7BLgYcXkSJrQwgEIBXq8XR0ePV6u7m5lVZVtYGYywzaa0LAoTkJUA+uXCY/06QftiQg6/lrzTjEiFnlnkymUOmbmPoi6LKX6oyxfsQNstrZ3G8/5rlfAZ55vkIQIzJWkIkALG22Jsf7u0dRRkOxJEQfF2WdVWpFjWHUsnHSqRR5Kbdnhwe7u/Ney8iqAZljsySjj1L557FgZMjos1mdXr62vX1q6ZZF0XJXOSZAAEAjuF3++YCoMratN0Y8jwfqvCIBvshkZHgAVBEdAIHAJglMiPRwNaqsqDvW+nu65N3QuisDz2ACKTq3BCJsQCoWkhM52zFoqhm8yOOPo+xx8hhOpmBpG72oNbXqK4xrW22+7NZshQgQ9qku1Pdfd+G0MUYRfJfjSFjjB+NJuPxbLtdTaf7nMYvBQBZxAeP5CADwKHpqD6ma0Voh+/n1sog+RZ1Qw1b1toYI2pTnOMwcqBB17dL6xygCb6zXe85dBw6ACFbFOWkKCsRAbDGYMoiAMY554o8vSYxRkKsq1Eark1wQfJOjxy5911ZuJODQ50ZzdXgrqX6vm+6rlmt7ptmo5NKZTmqqnGGzkXTrHd0IoAIIfiuayfTethkCDsmQwghRhbnKG+7XO8kpxNmwTRTKXq6VAgeABFNDD7HkWG+tjfGhOiD72y/eglhbd2IrJV+2zQ3fbU/nh7t1NwoEp2trS04n0MZY6iKwjkT4kMGHCoYZhXG+Eenp3qhmWwY0mjf9977drNZXl29YObRaGqt896vVvdd10wmc2uLvu9yYZ8ijzFmsbgJMeR6DREJsrwH0p+a6zn9vbIhGKdNmbqsAlmRp5veaOwjY32/yR0AyJ06iMH3vfd9a6G/o+kbtpo4VxpjgLt2ebFenI/njxSL6syoXtzQiYgxVpMJZEMMXEKmALn33WQ0GlVVH3PsZ8gT3RqqmtXq/vb2/PDw7ODgRFvq3vdVVd/dXW23ayK73a729493KmoEkKurV2U5ymMX6W8MDvQ8IjRtiw+q19QtT7GfNDaFQUSXNVKCCEVZL7f3vm/IFiJR9yXZKmyu2a25vSUQz/3K903wDUdv7Gh28l5pqVldh8jCKakrXy654yIiVVkMrYQU2R8YGxaO8+k07iRKgTRvrwO2bbtdLG7eeOPbr732bgihaTabzWqxuF2t7nXK7//t6YxVAAZhIBqNKRbp/39eh0Ih1EnQWqixgzZT9sBdDt5BmA9E+mtyBsAQLTGepeQQNqW8uqsxusBdH4eksphW1WHQ+dZekfnJUe/JSBsOyZHP6VLxGuiWVrBe0g6VP7XveuyAj8veAAAAAElFTkSuQmCC"
```

Comment:
I think it's better to truncate the Base64 output like as follows: "Base64Data": "iVBORw0KGgoAAAA...QmCC"
Displaying the entire three screen long value is awkward and has little or no practical use.